normandy/selfrepair/static/js/normandy_driver.js

```diff
@@ -123,10 +122,13 @@ let Normandy = {
             this.log(data);
             return Promise.resolve({});
         } else {
-            return xhr('POST', 'https://input.mozilla.org/api/v2/hb/', {
+            return fetch('https://input.mozilla.org/api/v2/hb/', {
+                method: 'POST',
                 data: data,
                 headers: {Accept: 'application/json'},
-            }).then(request => JSON.parse(request.responseText));
+            })
+            .then(response => response.json())
+            .then(heartbeatFlow => heartbeatFlow);
```

Comment:
Doesn't look like it o_O https://github.com/mozilla/normandy-actions/blob/master/actions/show-heartbeat/index.js#L73